### DISCUSSION TITLE
I'm still confused

### DISCUSSION THREAD

**Discussion by Teslimsama** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#25] [Outreachy 2022] [answered]
Please can someone please explain what supposed to be doing I have done subpage for open source what else

**Comment by BatoolMM** [comment, marked as answer]
Hi @Teslimsama, you have to apply all the feedback you received in your PR until it's merged. There are also few more tasks added every few days that you are welcome to try!
